api/v1alpha1/meshsync_types_test.go

```diff
@@ -94,28 +94,6 @@ var _ = Describe("The test case for the meshsync CRDs", func() {
 			Expect(meshSyncGet.Spec.Size == 5).Should(BeTrue())
 		})
 
-		It("The meshsync CRDs update the status of the resources", func() {
```

Comment:
Any reason for this to be removed?

Reply:
Nice catch, I check this one and add it back.

Reply:
@Azanul Can we merge this now?